src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -190,6 +241,35 @@ public void parse_validGitRepoLocations_repoConfigurationListCorrectSize() throw
         Assert.assertEquals(2, repoConfigs.size());
     }
 
+    @Test
+    public void parse_repoLocationsAndView_returnsLocationCliArguments() throws ParseException {
+        String input = String.format("-repos \"%s\" %s -view", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertTrue(cliArguments.isAutomaticallyLaunching());
```

Comment:
Let's do the assertion for `TEST_REPO_REPOSENSE` directory path here as well, since the test is both 'repo' and 'view'

Reply:
I have updated the test case, please have a look.